corehq/motech/models.py

```diff
@@ -94,7 +94,7 @@ class ConnectionSettings(models.Model):
         choices=api_auth_settings_choices,
     )
     username = models.CharField(max_length=255, null=True, blank=True)
-    password = models.CharField(max_length=255, blank=True)
+    password = models.CharField(max_length=1023, blank=True)
```

Comment:
Just wondering if we should not use `TextField` here? (I'm totally open to be convinced otherwise)

Reply:
I remember having a discussion around exactly this. @kaapstorm, can you remember why we ended up agreeing to use a char field instead of a text field here? I think it was because a text field would change how the UI is presented, however after testing it seems that the UI still remains the same after the type change.

Reply:
> I think it was because a text field would change how the UI is presented

That's exactly it. `TextField` is rendered with `textbox` instead of `input`. And in Postgres, `CharField` and `TextField` are the same column type.